portfolio/src/main/webapp/index.html

```diff
@@ -23,14 +23,26 @@ <h1>Andrew's Portfolio</h1>
       </button>
       <p id="hello-container"></p>
 
+    <h1>Cloud image storage</h1>
+    <form action="/upload" method="POST" enctype="multipart/form-data">
```

Comment:
nit: Currently on the site itself the organization/wording makes where to submit text vs images a bit confusing:
![image](https://user-images.githubusercontent.com/105953122/174691310-1baf3587-26be-4c55-b1b4-4a2eac4525c1.png)

Perhaps making them more separate or fusing them more coherently together would make it clearer to users